opampserver/pkg/sdkconfig/sdkconfig.go

```diff
@@ -80,10 +80,21 @@ func (m *SdkConfigManager) GetFullConfig(ctx context.Context, remoteResourceAttr
 		return nil, err
 	}
 
+	// We are moving towards passing all Instrumentation capabilities unchanged within the instrumentationConfig to the opamp client.
+	// Gradually, we will migrate the InstrumentationLibraryConfigs and SDK remote config into the instrumentationConfig and the agents to use it.
+	instrumentationConfig, err := configsections.GetWorkloadInstrumentationConfig(ctx, m.mgr.GetClient(), instrumentedAppName, podWorkload.Namespace)
```

Comment:
`instrumentationConfig` can be `nil` in case we got `apierrors.IsNotFound(err)`. in which case, the `FilterRelevantSdk` function below will panic when trying to access it